Documentation/markdown/SetupHelp.md

```diff
@@ -37,7 +48,8 @@ Before you can proceed to the Installation of **OpenInfraPlatform**, check, if t
 
 ![](./fig/CMake_Installation_settings.png)
 
-7. For detailed descriptions of all configuration options that can be selected in the CMake GUI, see [here](./CMakeOptions.md)
+7. For detailed descriptions of all configuration options that can be selected in the CMake GUI, see [here](./CMakeOptions.md) 
+	**HINT:** A few (red) warnings in the lower window of CMake can be ignored as long as it writes 'Configuring done' at the end
 8. After configuration process has successfully finished, click *Generate*.
```

Comment:
The hint 'A few (red) warnings ...' should be below point 8 'After configuration ...'
The red warnings come up during the configuration process. Sorry, I missed to point out the exact position.
[onec again, now my comment should be at the right place - including the last changes in the code]

Reply:
As we just discussed, in point 8 configuration process is already finished, so there wouldn’t be any need in this hint. 
We have decided, that this hint stays in point 7.